src/dropdown/dropdown.js

```diff
@@ -170,9 +162,53 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
         return;
       }
       dropdownCtrl.dropdownMenu = element;
+
+      var dropdown, appendToBody,
+          dropdownMenu = element;
+
+      scope.$watch(dropdownCtrl.isOpen, function( isOpen ) {
```

Comment:
I think I would favor this being in the controller - this adds an additional $watch for what seems to be unnecessary reasons.